plaidml/edsl/edsl.h

```diff
@@ -81,10 +82,15 @@ class Program {
  public:
   ///
   /// Program constructor
-  ///
-  Program(                                 //
-      const std::string& name,             //
-      const std::vector<Tensor>& outputs,  //
+  /// floatx is used to specify the underlying datatype of floating-point constants
+  /// and should be either DType::FLOAT32 or DType::FLOAT64.
+  /// intx is used to specify the underlying datatype of integer constants
+  /// and should be either DType::INT32 or DType::INT64.
```

Comment:
I think we should allow any floating point type for `floatx` and any integer type for `intx`.

Reply:
done